ccdImports/configFiles/CCD_Probate_Backoffice/CaseEventToFields.json

```diff
@@ -41,7 +41,7 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "solsNoOriginalWill", "PageFieldDisplayOrder": 3, "DisplayContext": "READONLY", "PageID": "solicitorUpdateProbatePage1", "PageDisplayOrder": 1, "FieldShowCondition": "willAccessOriginal=\"No\"", "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "willHasCodicils", "PageFieldDisplayOrder": 4, "DisplayContext": "MANDATORY", "PageID": "solicitorUpdateProbatePage1", "PageDisplayOrder": 1, "ShowSummaryChangeOption": "y"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "willNumberOfCodicils", "PageFieldDisplayOrder": 5, "DisplayContext": "MANDATORY", "PageID": "solicitorUpdateProbatePage1", "PageDisplayOrder": 1, "FieldShowCondition": "willHasCodicils=\"Yes\"", "ShowSummaryChangeOption": "y"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "dispenseWithNotice", "PageFieldDisplayOrder": 1, "DisplayContext": "MANDATORY", "PageID": "solicitorUpdateProbatePage2", "PageDisplayOrder": 2, "ShowSummaryChangeOption": "y"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "CaseEventID": "solicitorUpdateProbate", "CaseFieldID": "dispenseWithNotice", "PageFieldDisplayOrder": 1, "DisplayContext": "MANDATORY", "PageID": "solicitorUpdateProbatePage2", "PageDisplayOrder": 2, "ShowSummaryChangeOption": "y",  "CallBackURLMidEvent": "http://${CCD_DEF_CASE_SERVICE_BASE_URL}/case/sols-validate-executors"},
```

Comment:
just checking its ok for the scheme to be in url? Http? Would it not be https for prod?